python_banyan/banyan_base_multi/banyan_base_multi.py

```diff
@@ -32,6 +32,7 @@
 import msgpack_numpy as m
 import zmq
 import os
+import umsgpack
```

Comment:
I am not sure why you are importing umsgpack. The [msgpack package is being used ](https://github.com/MrYsLab/python_banyan/blob/4ff4b95c58cb0476d3363407f2ea161f8d061457/python_banyan/banyan_base_multi/banyan_base_multi.py#L31) currently in this file as all others. 
Is there a problem using msgpack for this file?

Reply:
umsgpack is used when self.numpy is false

https://github.com/MrYsLab/python_banyan/blob/4ff4b95c58cb0476d3363407f2ea161f8d061457/python_banyan/banyan_base_multi/banyan_base_multi.py#:~:text=default%3Dm.encode)-,else%3A,message%20%3D%20umsgpack.packb(payload),-pub_envelope%20%3D%20topic.encode

If if its not imported then these two tests fail (at least on my machine?) 
https://github.com/MrYsLab/python_banyan/blob/4ff4b95c58cb0476d3363407f2ea161f8d061457/python_banyan/tests/banyan_base_multi/test_banyanBaseMulti.py#:~:text=def%20test_publish_payload_invalid_socket(self,assert%20True